DRAFT_RELEASE_NOTES.md

```diff
@@ -100,11 +104,16 @@ extension-pkg-whitelist = "pydantic"
 ignore-patterns = '.*pb2[\S]*.py'
 ```
 
+### Upgrade Steps for Transitioning from Orphedomos to Fabric8
+If any extra configurations were added to the `orphedomos-maven-plugin` in addition to the generated defaults, executing the migration **will result in loss of these extra configurations**. To facilitate the upgrade, the following steps should be taken:
+1. Before executing the baton migration, it is recommended to ensure some form of version control is in place to preserve your existing `orphedomos-maven-plugin` configurations.
```

Comment:
looks like these bullets should be 1., 2., 3., instead of all 1's

Reply:
That is intentional. That way if we insert another point, we don't have to manually renumber. Markdown is smart enough to increment the numbers when the README is rendered

Reply:
oo good to know!